hera_pspec/pspecdata.py

```diff
@@ -819,3 +823,77 @@ def pspec(self, bls, beam=None, input_data_weight='identity', norm='I',
                     pairs.append((key1, key2))
                     
         return np.array(pvs), pairs
+
+    def rephase_to_dset(self, dset_index, inplace=True):
+        """
+        Rephase data in dsets to the LST grid of dset[dset_index] using
+        hera_cal.lstbin.lst_rephase.
```

Comment:
Please add a few simple details about what's happening here, e.g. are the LSTs being rephased to the centre of the reference LST bins?